### PR TITLE
Pxedeploy

### PR DESCRIPTION
**Pull request by maxiterr**

### CODE REVIEW

File: modules/network/finish/templates/etc/network/interfaces [hunk 1]
```diff
@@ -0,0 +1,13 @@
+auto lo
+iface lo inet loopback
+
+auto eth0
+iface eth0 inet static
+  address 192.168.2.15
+  netmask 255.255.255.0
+
+allow-hotplug wlan0
```
Comment (ggiamarchi) [inline modules/network/finish/templates/etc/network/interfaces]: I would variabilize the interface name `wlan0`

File: modules/network/finish/templates/etc/wpa_supplicant/wpa_supplicant.conf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
```
Comment (ggiamarchi) [inline modules/network/finish/templates/etc/wpa_supplicant/wpa_supplicant.conf:1]: Is this configuration vendor neutral ?

File: modules/pxedeploy-ubuntu-1404/finish/files/tftp/pxe/pxelinux.cfg/exemple-ubuntu-1404.menu [hunk 3]
```diff
@@ -0,0 +1,6 @@
+default install
+label install
+        menu label ^Install
+        menu default
+        kernel tftp://192.168.2.15/ubuntu/1404/linux
```
Comment (ggiamarchi) [inline modules/pxedeploy-ubuntu-1404/finish/files/tftp/pxe/pxelinux.cfg/exemple-ubuntu-1404.menu]: Where does this IP address comes from ?

Comment (ggiamarchi) [inline modules/pxedeploy-ubuntu-1404/finish/files/tftp/pxe/pxelinux.cfg/exemple-ubuntu-1404.menu]: Okay, got it ;) Static in `/etc/network/interfaces`.

File: modules/pxedeploy-ubuntu-1404/finish/files/var/www/pxe/ubuntu-1404.seed [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+d-i	passwd/username	string	ubuntu
+d-i	passwd/user-password	password	maxime
+d-i	passwd/user-password-again	password	maxime
+d-i	user-setup/allow-password-weak	boolean	true
+
+d-i	passwd/root-login	boolean true
+d-i	passwd/root-password	password	maxime
+d-i	passwd/root-password-again	password	maxime
+
+# set NTP !
```
Comment (ggiamarchi) [inline modules/pxedeploy-ubuntu-1404/finish/files/var/www/pxe/ubuntu-1404.seed]: Useless comment i guess.

File: modules/pxedeploy-ubuntu-1404/finish/files/tftp/pxe/pxelinux.cfg/exemple-ubuntu-1404.menu [hunk 5]
```diff
@@ -0,0 +1,6 @@
+default install
```
Comment (ggiamarchi) [inline modules/pxedeploy-ubuntu-1404/finish/files/tftp/pxe/pxelinux.cfg/exemple-ubuntu-1404.menu]: Why this file's called `example-...` ?

File: modules/pxedeploy/finish/templates/etc/dnsmasq.conf [hunk 6]
```diff
@@ -0,0 +1,7 @@
+interface=eth0
+dhcp-boot=pxelinux.0
+enable-tftp
+tftp-root=/tftp/pxe/
+dhcp-range=192.168.2.200,192.168.2.250,255.255.255.0,12h
+dhcp-option=3,192.168.2.1
```
Comment (ggiamarchi) [inline modules/pxedeploy/finish/templates/etc/dnsmasq.conf]: DHCP options 3 and 6 not necessary at this point.

File: modules/network/finish/templates/etc/network/interfaces (hunk 1 above)
Comment (maxiterr) [inline modules/network/finish/templates/etc/network/interfaces]: Fixed

File: modules/network/finish/templates/etc/wpa_supplicant/wpa_supplicant.conf (hunk 2 above)
Comment (maxiterr) [inline modules/network/finish/templates/etc/wpa_supplicant/wpa_supplicant.conf:1]: I guess

File: modules/pxedeploy-ubuntu-1404/finish/files/var/www/pxe/ubuntu-1404.seed (hunk 4 above)
Comment (maxiterr) [inline modules/pxedeploy-ubuntu-1404/finish/files/var/www/pxe/ubuntu-1404.seed]: Fixed

File: modules/pxedeploy-ubuntu-1404/finish/files/tftp/pxe/pxelinux.cfg/exemple-ubuntu-1404.menu (hunk 5 above)
Comment (maxiterr) [inline modules/pxedeploy-ubuntu-1404/finish/files/tftp/pxe/pxelinux.cfg/exemple-ubuntu-1404.menu]: Fixed. change name to `ubuntu-1404`

File: modules/pxedeploy/finish/templates/etc/dnsmasq.conf (hunk 6 above)
Comment (maxiterr) [inline modules/pxedeploy/finish/templates/etc/dnsmasq.conf]: DHCP option has been remove

**Pull request merged**